.travis.yml

```diff
@@ -121,33 +129,27 @@ before_script:
       --project phenomenal-1145 -q
 
 script:
-  # Packer -force for Azure and Openstack does not automatically remove the artifacts from previous build.
```

Comment:
In the previous stable version (i.e. the current master), before running the `packer build` command we were checking only for Azure and Openstack whether or not there were any duplicates (because the related packer builders do not automatically remove namesake artifacts from previous builds.) The same logic is still present, but it needed to be moved in the `after_success` stage the image's building process in order to take into account the scenario of the cronjob script for security updates. This decision is based on the fact that the cronjob logic considers updating an already existing kubenow stable, instead of building from scratch and check any security updates. However, if we clean the image before running the packer build command, then packer will certainly fail.